anjali/sb-pt-cont/8.js

```diff
@@ -0,0 +1,131 @@
+// https://levelbuilder-studio.code.org/s/sb-pt-cont/lessons/1/levels/8
+/*
+Add at least two other sprites to your Story Board. Give each sprite a unique costume and location!
+*/
+if (!validationProps.successCriteria) {
+  validationProps.successCriteria = {
+    threeSprites: false,
+    differentCostumes: false,
+    differentLocations: false
+  };
+}
+
+if (!validationProps.previous) {
+  validationProps.previous = {
+  	waitTime: 120
```

Comment:
Why is this in `validationProps.previous`?

Reply:
It's a system I set up to delay the bar more and more as new events happen so the student can see all the events play out.